ivy/functional/ivy/general.py

```diff
@@ -1845,20 +1846,20 @@ def einops_reduce(
 
     Examples
     --------
-    With :class:`ivy.Array` input:
```

Comment:
the original `input` should be fine here :)

Reply:
Resolved it @juliagsy